r/tests/testthat/test-dplyr-summarize.R

```diff
@@ -18,6 +18,7 @@
 skip_if(on_old_windows())
 
 withr::local_options(list(arrow.summarise.sort = TRUE))
+old_options <- options(rlib_warning_verbosity = "verbose")
```

Comment:
using `options` directly totally works, but since we have a call to `withr::local_options()` here already, does it work if we put `rlib_warning_verbosity = "verbose"` inside that list?

Reply:
We're also raising 10 or so warnings here that 

```r
`summarise()` has grouped output by 'some_grouping'. You can override using the `.groups` argument.
```

Do we want to suppress those as well while we're at it?

Reply:
Could we assert the message once and then wrap in `suppressMessages()` like we do for the warnings?

Reply:
Wait we could set an option for this too instead. From `?dplyr::summarize`:

```
In addition, a message informs you of that choice, unless the result is ungrouped, the option "dplyr.summarise.inform" is set to FALSE, or when summarise() is called from a function in a package.
```

Reply:
Oh nice, yeah maybe do a local option there setting it to false with a comment as to why | what the message is that we are suppressing